doc/btp_gmcs.txt

```diff
@@ -0,0 +1,70 @@
+Generic Media Control Service(ID 23)
+=================================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read supported commands
+		Controller Index:       <controller id>
+		Command parameters:     <none>
+		Response parameters:    <supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Command Send
+
+		Controller Index:	<controller id>
+		Command parameters:	Opcode (1 octets)
+					Use Param (1 octet)
```

Comment:
`Use Param` is boolean or data length of `Param`? Please add a little annotation here.

Reply:
Use param is a boolean